apps/emqx_exhook/test/emqx_exhook_demo_svr.erl

```diff
@@ -274,10 +274,10 @@ on_session_discarded(Req, Md) ->
     %io:format("fun: ~p, req: ~0p~n", [?FUNCTION_NAME, Req]),
     {ok, #{}, Md}.
 
--spec on_session_takeovered(emqx_exhook_pb:session_takeovered_request(), grpc:metadata())
+-spec on_session_takenover(emqx_exhook_pb:session_takenover_request(), grpc:metadata())
```

Comment:
I couldn't find this `emqx_exhook_pb` module.  Is it generated by `exhook.proto` or something?

Reply:
yes, it's generated